tests_integration/test_api_integration.py

```diff
@@ -0,0 +1,288 @@
+import unittest
+from os import path
+
+from tests_integration import tests_integration
+
+import api
+import model
+
+
+path_to_module = path.dirname(__file__)
+if len(path_to_module) == 0:
+    path_to_module = '.'
+
+
+class TestApiIntegration(unittest.TestCase):
+
+    def setUp(self):
+        print("\nStarting " + self.__module__ + ": " + self._testMethodName)
+
+    def test_send_and_get_project(self):
+        """
+        Tests sending and receiving project data
+        :return:
+        """
+        test_api = api.ApiCalls(
```

Comment:
The main reason was because the old uploader had issues with spinning off multiple instances of the api, but still cached things between them. The new uploader doesn't have this problem, but I didn't think to switch how the tests were set up. I moved the api initialization to the setup now.